src/data_storage/import/container_import.ts

```diff
@@ -444,8 +444,8 @@ export default class ContainerImport {
             return
           }
         }
-        
-        resolve(Result.Success('Ontology creation is complete! Container ID: ' + containerID))
+
+        resolve(Result.Success(containerID))
```

Comment:
this was changed so that the endpoint response could be used to quickly query the container